tests/epyccel/test_arrays.py

```diff
@@ -4027,7 +4027,15 @@ def test_array_float_nested_C_array_initialization_3(language):
 #==============================================================================
 # NUMPY SUM
 #==============================================================================
-
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.xfail(reason="List indexing is not yet supported in C"),
```

Comment:
Please reference the issue that will fix this (this makes it easier to find all xfails that should be removed when you fix the issue as you just need to search for the number)